NodeManagerLibrary.ino

```diff
@@ -3972,34 +3978,51 @@ void SensorNeopixel::setColor(char* string) {
   long color = 0;
   char * p = strstr(string, ",");
   if (p){ 
-    char pixelnum[6];
+    char pixelnum[10];
     int pos = (int) (p - string);
+    if (pos >= 10)
+      return;
     strncpy(pixelnum, string, pos);
     pixelnum[pos] = 0;
 
-    int pixel_num = atoi(pixelnum);
-
-    color = strtol(string + pos+1, NULL, 16);
+    int pixel_num = 0;
```

Comment:
Would you mind adding some comments in this routine when parsing the received string? Just to give other users willing to customize the behavior to understand the logic. Thanks!

Reply:
added